uint8 NONE=255
uint8 TAKE_OBJECT=0
uint8 PUT_OBJECT=1
uint8 STOP=2

uint8 LEFT=3
uint8 RIGHT=4
uint8 TOP=5
uint8 BEHIND=6
uint8 FRONT=7

uint8 command
Shape source_block
uint8 relative_to
Shape target_block
